src/prefect/environments/storage/docker.py

```diff
@@ -464,7 +464,7 @@ def create_dockerfile_object(self, directory: str) -> str:
         if self.env_vars:
             white_space = " " * 20
```

Comment:
This huge whitespace is odd - if the point is to align the set environment variables, 

```suggestion
            white_space = " " * 4
```

would make more sense

Reply:
I think they actually weren't aligned at all -- I'll commit this then take a look locally